bot/exts/code_jams/_views.py

```diff
@@ -0,0 +1,136 @@
+    async def announce(self, button: discord.ui.Button, interaction: discord.Interaction) -> None:
+        """A button to send the announcement."""
+        button.label = "Teams have been announced!"
+        button.disabled = True
+        self.cancel.disabled = True
+        self.stop()
+        await interaction.response.edit_message(view=self)
+        announcements = self.guild.get_channel(Channels.summer_code_jam_announcements)
+        await announcements.send(
+            f"<@&{Roles.code_jam_participants}> ! You have been sorted into a team!"
+            " Click the button below to get a detailed description!",
+            view=JamTeamInfoView(self.bot)
+        )
+
+    async def interaction_check(self, interaction: discord.Interaction) -> bool:
+        """Global check to ensure that the interacting user is the user who invoked the command originally."""
+        if interaction.user != self.original_author:
+            await interaction.response.send_message(
+                ":x: You can't interact with someone else's response. Please run the command yourself!",
+                ephemeral=True
+            )
+            return False
+        return True
+
+
+class JamTeamInfoView(discord.ui.View):
+    """A persistent view to show Team related data to users."""
+
+    def __init__(self, bot: 'SirRobin'):
+        super().__init__(timeout=None)
+        self.bot = bot
+
+    @discord.ui.button(label="Show me my team!", style=discord.ButtonStyle.blurple, custom_id="CJ:PERS:SHOW_TEAM")
+    async def show_team(self, button: discord.ui.Button, interaction: discord.Interaction) -> None:
+        """A button that sends an ephemeral embed with the team's description."""
+        try:
+            team = await self.bot.code_jam_mgmt_api.get(f"users/{interaction.user.id}/current_team",
+                                                        raise_for_status=True)
+        except ResponseCodeError as err:
+            if err.response.status == 404:
+                await interaction.response.send_message("It seems like you're not a participant!", ephemeral=True)
+            else:
+                await interaction.response.send_message(
+                    "Something went wrong while processing the request! We have notified the team!",
+                    ephemeral=True
+                )
+                log.error(err.response)
+        else:
+            response_embed = discord.Embed(
+                title=f"You have been sorted into {team['team']['name']}",
+                colour=discord.Colour.og_blurple()
+            )
+            team_channel = f"<#{team['team']['discord_channel_id']}>"
+            team_members = [f"<@{member['user_id']}>" for member in team["team"]["users"]]
+            response_embed.add_field(name="Your team's channel:", value=team_channel)
+            response_embed.add_field(name="Your team's members:", value="\n".join(team_members))
```

Comment:
For me "Your team members:" sound better, but not 100% sure.

Reply:
You're right, maybe "teammates?"